perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AttributesManagerMethod.java

```diff
@@ -241,10 +241,18 @@
 			} else if (parms.contains("resource")) {
 				if (parms.contains("member")) {
 					if (parms.contains("workWithUserAttributes")) {
-						return ac.getAttributesManager().getAttributes(ac.getSession(),
-								ac.getResourceById(parms.readInt("resource")),
-								ac.getMemberById(parms.readInt("member")),
-								parms.readBoolean("workWithUserAttributes"));
+						if (parms.contains("attrNames")) {
+							return ac.getAttributesManager().getAttributes(ac.getSession(),
+									ac.getResourceById(parms.readInt("resource")),
+									ac.getMemberById(parms.readInt("member")),
+									parms.readList("attrNames", String.class),
```

Comment:
No, it doesn't since existing javadoc is for version without ``attrNames`` parameter. Also I believe your implementation also differs in fact, that you return also user_facility attributes, which is not case of original method (member-resource, member, user).